drydock/templates/kustomized/global-stack/kustomization.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: {{ DRYDOCK_GLOBAL_STACK_NAMESPACE }}
+
+{%- if DRYDOCK_GLOBAL_RESOURCES %}
+resources:
+  {{ patch("drydock-global-resources") }}
+{%- endif %}
+
+helmCharts:
+- name: kube-prometheus-stack
+  repo: https://prometheus-community.github.io/helm-charts
+  version: {{ DRYDOCK_PROMETHEUS_VERSION }}
+  releaseName: prometheus
+  includeCRDs: true
+{%- if DRYDOCK_PROMETHEUS_VALUES %}
+  valuesInline:
+    {{ DRYDOCK_PROMETHEUS_VALUES | indent(4) }}
+{%- endif %}
+{{ patch("drydock-global-helm-charts") }}
+
+{%- if DRYDOCK_PROMETHEUS_PATCHES %}
+patches:
+{{ DRYDOCK_PROMETHEUS_PATCHES }}
+{%- endif %}
```

Comment:
```suggestion
patches:
{{ DRYDOCK_PROMETHEUS_PATCHES }}
```

Reply:
but what if `DRYDOCK_PROMETHEUS_PATCHES` is empty?

Reply:
> but what if `DRYDOCK_PROMETHEUS_PATCHES` is empty?

Nothing